.github/workflows/build.yml

```diff
@@ -36,8 +45,29 @@ jobs:
       with:
         name: Linux flatc binary ${{ matrix.cxx }}
         path: flatc
+    # Below if only for release.
+    - name: Zip file
+      if: startsWith(github.ref, 'refs/tags/')
+      run: zip Linux.flatc.binary.${{ matrix.cxx }}.zip flatc
+    - name: Release zip file
+      uses: softprops/action-gh-release@v1
+      if: startsWith(github.ref, 'refs/tags/')
+      with:
+        files: Linux.flatc.binary.${{ matrix.cxx }}.zip
+    - name: Generate SLSA subjects - clang
+      if: matrix.cxx == 'clang++-12' && startsWith(github.ref, 'refs/tags/')
+      id: hash-clang
+      run: echo "::set-output name=hashes::$(sha256sum Linux.flatc.binary.${{ matrix.cxx }}.zip | base64 -w0)"
+    - name: Generate SLSA subjects - gcc
```

Comment:
Do we need both of these separated? I thought the the `strategy` labels would handle this.

Reply:
Unfortunately we do. I don't like it either. The reason it's needed is because we collect all digests in the `release-digests` job, and if we use the same name they corrupt each other. GitHub automatically creates 2 different jobs for each `strategy`, but the `release-digests` needs to differentiate the result from each of them.

Reply:
Thanks.